### PR TITLE
Add TeV catalog to gamma-sky.net

### PR DESCRIPTION
**Pull request by vorugantia**
In this PR, I've mainly been working to add our new TeV catalog (from gamma-cat) to the website. This is what I've done:

* Load new TeV catalog in `catalogs.py`
* Show TeV catalog in `MapViewComponent`
* Show new data in the detailed view in `CatViewComponent`

The app is running perfectly. However, I ran into some problems along the way that I need to fix later (either in this PR or another one):

* Right now, in `catalogs.py`, I cannot dump data from all columns of the TeV catalog and convert it to JSON with pandas. Here is the error message I get:

```
ValueError: Cannot convert a table with multi-dimensional columns to a pandas DataFrame.
```

Because of this, I haven't added distance and spectral info to the detailed view in `CatViewComponent`. Right now it just displays the names and positions of each source. I'm not sure how to work around the error. Maybe I should just ignore all multi-dimensional columns for now?

* I've started to split up the one popup component into four separate components, each one containing a unique HTML template for their respective catalog. Although I'm halfway there, and the code works as-is, configuring the popup in `MapComponent` still looks very messy and should be fixed.

@cdeil - Can you take a look at the code when you're free to see if I'm headed in the right direction? Also, do you have a solution for the Python error I mentioned above? After this PR is closed we should be ready to publish the new website.

### CODE REVIEW

File: gammasky/catalogs.py [hunk 1]
```diff
@@ -10,50 +10,92 @@
...
+        'source_id',
+        'common_name',
+        'gamma_names',
+        'other_names',
+        'ra',
+        'dec',
+        'glon',
+        'glat',
+    ]
+    # cols = table.colnames  <-- Not working right now because I get the error: Cannot convert a table with multi-dimensional columns to a pandas DataFrame.
```
Comment (vorugantia) [inline gammasky/catalogs.py]: This causes the Python error. Instead I pass only the name and position columns into pandas.

Comment (adonath) [reply gammasky/catalogs.py]: gamma-cat includes multi-dimensional columns for the SED data. As a temporary fix you could  exclude those e.g. using  `cols = [name for name in table.colnames if not 'sed' in name]`

Comment (cdeil) [reply gammasky/catalogs.py]: You could also read https://github.com/astropy/astropy/issues/4604#issuecomment-184425733 to get some info on Astropy table, pandas Dataframe and JSON serialisation.

Short answer: to drop all multi-dimensional columns you can use this code snippet to make a list of columns to keep, and then index into the table with that list of colnames:
```python
cols = tuple(name for name in table.colnames if len(table[name].shape) <= 1)
t2 = table[cols]
```

**Comment by cdeil** [comment]
@vorugantia - This sounds great!

And yes, we should deploy a new version of the website once gamma-cat is added.
I don't know if it's a good idea to post to ArXiv during the holidays, it might get lost in people's inboxes, and posting in January is smarter.

Unfortunately I don't have time for code review before the holidays.
(HESS survey paper goes to collaboration circulation tomorrow.)

So my suggestion would be that if you  have something working, merge it.
And if you have specific other questions or things I should look at, ping me again.

File: gammasky/catalogs.py (hunk 1 above)
Comment (vorugantia) [reply gammasky/catalogs.py]: Okay, I have it working now. Thanks @adonath and @cdeil!

**Comment by vorugantia** [comment]
* I'm now dumping all of the fits file except for SED (multidimensional) data into the pandas dataframe.

* Popups fully reconfigured - code is much cleaner now.

Merging this PR...

**Comment by cdeil** [comment]
Thanks!

I tried this locally just now and it works for me.

**Pull request merged**